openedx.yaml

```diff
@@ -0,0 +1,7 @@
+# This file describes this Open edX repo, as described in OEP-2:
+# http://open-edx-proposals.readthedocs.io/en/latest/oeps/oep-0002.html#specification
+
+nick: edx_openid
```

Comment:
The spec for these files recommends a max of 4 characters for this; maybe just `opid`?